### DISCUSSION TITLE
avatar database provider?

### DISCUSSION THREAD

**Discussion by ghost** [vrcx-team/VRCX#331] [General]
hi i was just looking into the settings and seen a option for "avatar database provider" it asks for a url im just curious what info is that looking for? a json file? if so any specific format or what?

**Comment by Natsumi-sama** [comment]
https://gist.github.com/RequiDev/d209f79c6d663dbd5c1428ebbe2c944a

**Reply by Chillstice** [reply]
> https://gist.github.com/RequiDev/d209f79c6d663dbd5c1428ebbe2c944a

This project has been terminated.

Are there any alternatives? I'm not finding anything anymore.

**Reply by Myrkie** [reply]
you would have to host your own

**Reply by Myrkie** [reply]
I host my own for friends as its not on a dedicated server so it can get killed by lots of requests.
